test/unittest/utils/IPLocatorTests.cpp

```diff
@@ -1081,6 +1283,13 @@ TEST(LocatorListComparison, locatorList_comparison)
     locator_list_1.push_back(locator);
 
     ASSERT_FALSE(locator_list_1 == locator_list_2);
+
```

Comment:
You should clean both lists before checking this. The lists are already different so adding new locators does not change anything over the features already tested.

Reply:
First case is on purpose to reach a specific line in coverage.
Second case however is true and it has been fixed.